dev/src/types.ts

```diff
@@ -321,6 +321,15 @@ export interface ValidationOptions {
   allowTransforms: boolean;
 }
 
+/**
+ * An options object that can be used to toggle the default rate limit
+ * throttling implemented by BulkWriter.
+ */
+export interface BulkWriterOptions {
+  /** Whether to disable throttling as specified by the 500/50/5 rule. */
```

Comment:
I would drop the mention of 500/50/5 here. Details are not relevant here since we explicitly don't want to throttle.

Reply:
done.